packages/manager/modules/pci/src/projects/project/storages/blocks/block/edit/edit.html

```diff
@@ -1,57 +1,68 @@
-<div data-ng-if="$ctrl.loading" class="text-center">
-    <oui-spinner ></oui-spinner>
+<div class="p-5">
+    <div data-ng-if="$ctrl.loading" class="text-center">
+        <oui-spinner ></oui-spinner>
```

Comment:
```suggestion
        <oui-spinner></oui-spinner>
```

Reply:
Should be managed by a linter, we don't need human lint -> It takes a lot of time and I don't find it relevant to have some comments on this topic in a PR.

Can we try to improve htmlhint maybe and purpose a new rule?

Reply:
> It takes a lot of time and I don't find it relevant to have some comments on this topic in a PR

Not really because it will be trapped at the review. It doesn't require too much effort.

Both tool HTMLHint and XML validator don't complain because it still seen as a valid document.

Otherwise theses snippets can help in certain way:
- https://github.com/antleblanc/atom-ovh-ui-kit-snippets
- https://github.com/antleblanc/vscode-ovh-ui-kit-snippets

Other long term proposal is that we could search for a whitespace sensitive syntax for writing HTML
such as [Pug](https://github.com/pugjs/pug) for instance.